doc/content/sfr/subchannel/thors/thors.md

```diff
@@ -56,19 +56,23 @@ THORS bundle 5B has the same fuel configuration as bundle 2B, except that 0.0711
 
 ## Results for edge blockage
 
-The first case presented here is the high flow case (FFM Series 6, Test 12, Run 101). The thermocouples are located at the middle of the exit region. There is a subchannel index correspondence between the Figure [fig:thors2] and the SCM model shown in Figure[fig:hex_index] as follows: 34(39), 33(38), 18(20), 9(19), 3(4), 0(1), 12(11) and 25(30). The number outside the parentheses refers to the SCM model, and the number inside the parentheses refers to the experimental convention. SCM calculation along with the experimental measurements are shown in Figure [fig:FFM-5B_high]. The code calculations exhibits generally good agreement with the experimental measurements except for the the edge subchannels ($34,33,18$) which is likely due to the model not accurately replicating the flow area there. SCM uses an assembly-wide constant wire diameter, while in the experimental assembly the wires at the edge subchannels had half the diameter. Also the high flow case has higher turbulence which would induce higher mixing.
+The case presented here is the high flow case (FFM Series 6, Test 12, Run 101). The thermocouples are located at the middle of the exit region. There is a subchannel index correspondence between the Figure [fig:thors2] and the SCM model shown in Figure[fig:hex_index] as follows: 34(39), 33(38), 18(20), 9(19), 3(4), 0(1), 12(11) and 25(30). Where the number outside the parentheses refers to the SCM model and the number inside the parentheses, refers to the experimental convention. SCM calculation along with the experimental measurements is shown in Figure [fig:FFM-5B_high]. The code calculations excibits generally good agreement with the experimental measurements. The least agreement occurs at the edge subchannels ($34,33$) which is likely due to the model not accuretally replicating the flow area there. SCM uses an assembly-wide constant wire diameter, while in the experimental assembly the wires at the edge subchannels had half the diameter.
```

Comment:
please fix the spelling back

accurately
exhibits
is -> are
...

you should update the source of this documentation to have correct spelling